strategy.ini

```diff
@@ -0,0 +1,88 @@
+; some useful links:
+; - https://janelia-flyem.github.io/licenses.html
+; - https://dwheeler.com/essays/floss-license-slide.html
+
+; Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
+	; aliases for MIT License
+	MIT
+	MIT license
+	https://opensource.org/licenses/MIT
+	License :: OSI Approved :: MIT
+
+	; aliases for BSD License (and variants)
+	BSD
+	BSD license
+	new BSD
+	(new) BSD
+	new BDS license
+	simplified BSD
+	3-Clause BSD
+	BSD-3-Clause
+	BSD 3-Clause
+	BSD-2-Clause
+	BSD-like
+
+	; aliases for Apache License version 2.0
+	Apache 2.0
+	Apache License 2.0
+	Apache License, Version 2.0
+	Apache License Version 2.0
+	ASL 2
+;	some packages use 'Apache Software' as license string,
+;	which is ambiguous. However, 'Apache Software'
+;	will likely match with 'Apache 2.0'
+	Apache Software
+	BSD, Public Domain, Apache
+	http://www.apache.org/licenses/LICENSE-2.0
+
+;	PSF (BSD-style)
+	Python Software Foundation
+	PSF
+
+	; other permissive licenses
+	Historical Permission Notice and Disclaimer (HPND)
+	HPND
+	ISC
+	BSD or Apache License, Version 2.0
+	Modified BSD
+	Expat
+
+unauthorized_licenses:
+;	aliases for MPL 2.0
```

Comment:
citing [https://www.mozilla.org/en-US/MPL/2.0/FAQ/](https://www.mozilla.org/en-US/MPL/2.0/FAQ/)

> Q2: Why yet another open source license?
> The MPL fills a useful space in the spectrum of free and open source software licenses, sitting between the Apache license, which does not require modifications to be shared, and the GNU family of licenses, which requires modifications to be shared under a much broader set of circumstances than the MPL.

Hence, MPL is less permissive than Apache 2.0.